docs/get_started/using/index.md

```diff
@@ -195,6 +195,24 @@ Simple interactions with the `volumes` endpoint. Refer to the [volumes](../../ap
 {}
 ```
 
+## Paginate volumes
+
```

Comment:
There should be a description here. What about:

"Certain API resources support limits and means to paginate objects to perform operations in batches. The below example uses the `volumes` resource to paginate volume objects."

Reply:
All object sets support 'limits' which should be used for pagination.

Reply:
Updated the PR to address this.

Reply:
LGTM